tests/streamlit/__snapshots__/test_commands.ambr

```diff
@@ -11,38 +11,6 @@
   
   '''
 # ---
-# name: test_deploy_put_files_on_stage[example_streamlit-merge_definition1]
```

Comment:
Was this removed by your change?

Reply:
It was a leftover snapshot- not used by any test